### PR TITLE
WIP: Test refactor

### PR DESCRIPTION
**Pull request by Lorquas**
This branch will be used to develop the new tests

Downstream pytests will be pushed upstream (here) and triggered by packit/tmt

### CODE REVIEW

File: systemtest/conftest.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from dynaconf import Dynaconf
+from pytest import fixture
+import os
+
```
Comment (jirihnidek) [inline systemtest/conftest.py:4]: No doc string explaining purpose of this file.

File: systemtest/conftest.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+current_directory = os.path.dirname(os.path.realpath(__file__))
+
+@fixture
+def settings():
```
Comment (jirihnidek) [inline systemtest/conftest.py:8]: No doc string explaining purpose of this method.

**Review by jirihnidek — COMMENTED** [review]
I know that it is only draft, but I have several requests anyway:

1. Please add as many doc strings as possible and try to describe your code, because current code does not make almost any sense to me.
2. Please use type hints. It helps readability of code and it avoid making mistakes
3. Please describe your commit with descriptive log message.
4. Please try to use some code linter. There is too many issues.

I believe it is better to adopt some good coding habit sooner than later.

**Comment by Lorquas** [comment]
@jstavel please work on this in my fork: https://github.com/Lorquas/rhc/pull/1  . I've invited you as a contributor.  This PR has been downgraded to simply working out which accounts we need to get packer and copr builds working.  Once that fork has good code we will shape it up for an entirely new PR against this repo.   This PR will not be merged and closed once the accounts are working.
@jirihnidek sorry for the confusion.  I will make sure doc strings and type hinting are added to the PR.

**Comment by jstavel** [comment]
Hi all. I will close this PR. I will start contributing to the fork from Lorquas.

**Pull request closed without merging**